upgrade-notes/backend_20250214_123809.md

```diff
@@ -0,0 +1,6 @@
+#### remove no longer necessary image migration for proxy ([#3801](https://github.com/shopsys/shopsys/pull/3801))
+
+- if you are upgrading from Shopsys Platform 14.0 or newer, you do not need to do any other steps
+- if you are upgrading from Shopsys Platform 13.0 or older, you need to check this PR and do the migration of images folder manually
+    - note: in version 14.0 we have changed image folder structure and introduced migration command to easily switch to this new structure, after three versions we are removing this command as it was one time action and is no longer necessary
```

Comment:
these upgrade notes will be for the `17.0` version, there is no way anyone upgrading from any other version than `16.0` so I am not sure if these notes make any sense :thinking: 

The only thing I would put here would be the removal of `MigrateImagesCommand`, `IMAGE_STRUCTURE_MIGRATED_FOR_PROXY` setting, and `migrate-images-for-proxy` phing target as they are no longer necessary and I would add an instruction to remove their `Version20231020173331` entry from the migrations lock file :slightly_smiling_face:

Reply:
I have added those mentions and I have also keeped the text as everyone can continue without need to do anything, because they have been upgrading from version >= 14.0, but you never know as there might be someone dedicated to upgrade theirs project all the way from version 7 :D I like to keep it safe.